Code/Framework/AzCore/AzCore/Debug/Budget.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Debug/BudgetTracker.h>
+#include <AzCore/Math/Crc.h>
+
+namespace AZ::Debug
+{
+    // A budget collates per-frame resource utilization and memory for a particular category
+    class Budget final
+    {
+    public:
+        explicit Budget(const char* name);
+        Budget(const char* name, uint32_t crc);
```

Comment:
Actually you mark both constructors `constexpr`, since the all the Crc32 constructors are also `constexpr`.
`AZ_CRC_CE` doesn't have to be used. The regular Crc32 constructor can be.

Reply:
Thanks for the suggestion, but will probably leave non-constexpr right now (it only looks like it can be constexpr because the class itself is a stub class)